lib/util/escapeRegExp.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Escapes RegExp special characters from a string
```

Comment:
or maybe you're suggesting that we should only support `*userInput*` and don't provide the user with a way to use wildcards?